docs/src/components/ConfigurationDrawer.tsx

```diff
@@ -53,8 +57,16 @@ export const ConfigurationDrawer = ({
   isSummaryToggleVisible,
   setIsSummaryToggleVisible,
   isSummaryToggleInitiallyEnabled,
-  setIsSummaryToggleInitiallyEnabled
+  setIsSummaryToggleInitiallyEnabled,
+  rerankingConfigurationString,
+  setRerankingConfigurationString
 }: Props) => {
+  const rerankingConfigPlaceholder = `{
+  "rerankerId": 272725718,
```

Comment:
I added this placeholder to show an example, but also to indicate that they need to quote the property name. You shouldn't need to in real usage, but parsing this for use as a component prop was causing errors without quotes, e.g.

```
{
  rerankerId: 272725718,
  mmrConfig: {
    "diversityBias": 0
  }
```